scrapyrt/resources.py

```diff
@@ -212,6 +212,13 @@ def prepare_crawl(self, api_params, scrapy_request_args, *args, **kwargs):
             max_requests = api_params['max_requests']
         except (KeyError, IndexError):
             max_requests = None
+
+        crawler_params = api_params.copy()
+        for api_param in ['max_requests', 'start_requests', 'spider_name',
+                          'url']:
```

Comment:
I see we also have documented api param "callback", should this param be popped here too? What do you think about making list of documented params as attribute to crawl resource? So that it is visible at the top and we can reuse it somewhere else, it will also be easier to maintain in case we add some new arguments that wont be passed to crawler